doc/classes/GDExtension.xml

```diff
@@ -1,49 +1,64 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="GDExtension" inherits="Resource" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		A shared native library for GDExtension.
 	</brief_description>
 	<description>
+		The [GDExtension] resource type represents a set of libraries written for GDExtension. These libraries expand the existing functionality of the engine. They are handled by the [GDExtensionManager] singleton.
```

Comment:
While the `.gdextension` file will list multiple libraries, a `GDExtension` resource will only ever represent a single one. Maybe:

```suggestion
		The [GDExtension] resource type represents a libraries written for GDExtension, which can expand the functionality of the engine. The [GDExtensionManager] singleton is responsible for loading, reloading and unloading [GDExtension] resources.
```